public_html/wp-content/themes/wporg-events-2023/postcss/blocks/event-list.pcss

```diff
@@ -0,0 +1,106 @@
+.wporg-marker-list__container {
+	padding-left: 0;
+
+	.wporg-marker-list-item {
+		border: 1px solid var(--wp--preset--color--light-grey-1);
+		border-bottom: none;
+		padding: var(--wp--preset--spacing--20);
+		list-style: none;
+		font-size: var(--wp--preset--font-size--small);
+
+		&:first-child {
+			border-radius: 2px 2px 0 0;
+		}
+
+		&:last-child {
+			border-radius: 0 0 2px 2px;
+			border-bottom: 1px solid var(--wp--preset--color--light-grey-1);
+		}
+
+		@media (--large) {
+			display: grid;
+			align-items: start;
+			gap: var(--wp--preset--spacing--20);
+			grid-template-columns: 45% 1fr 2fr;
+		}
+
+		@media (--xlarge) {
+			font-size: var(--wp--preset--font-size--normal);
+		}
+
+		@media (--wide) {
+			grid-template-columns: 45% 1fr 1.5fr;
+		}
+
+		@media (--huge) {
+			grid-template-columns: 55% 1fr 1fr;
+		}
+
+		@media (--xhuge) {
+			grid-template-columns: 60% 1fr 1fr;
+		}
+
+		.wporg-marker-list-item__title {
+			margin: 0;
+			font-family: var(--wp--preset--font-family--inter);
+			font-size: var(--wp--preset--font-size--small);
+			line-height: var(--wp--custom--body--typography--line-height);
+			--wp--preset--spacing--30: 0;
+
+			&.a {
```

Comment:
This has caused a regression where the titles are underlined. They should not be.

Reply:
Thanks! It's fixed [here](https://github.com/WordPress/wordcamp.org/pull/1181/commits/4284895341927035e10a517087538569bf97b68f).